cypress/utils/apps/explore/constants.js

```diff
@@ -3,8 +3,39 @@
  * SPDX-License-Identifier: Apache-2.0
  */
 
+export const DATASOURCE_NAME = 'data-logs-1';
+export const START_TIME = 'Jan 1, 2020 @ 00:00:00.000';
+export const END_TIME = 'Jan 1, 2024 @ 00:00:00.000';
+
+export const clusterName = 'test_cluster';
+export const clusterConnection = 'http://localhost:9200';
+
+export const S3_CLUSTER = Cypress.env('S3_ENGINE') || {};
+
+export const DS_API_PREFIX = '/api/saved_objects';
+export const DS_API = {
+  DATA_SOURCES_LISTING: `${DS_API_PREFIX}/_find?fields=id&fields=description&fields=title&per_page=10000&type=data-source`,
+  CREATE_DATA_SOURCE: `${DS_API_PREFIX}/data-source`,
+  DELETE_DATA_SOURCE: `${DS_API_PREFIX}/data-source/`,
+};
+export const DSM_API = '/internal/data-source-management/fetchDataSourceMetaData';
+
+export const BASE_QUERY_ENHANCEMENTS_API = '/api/enhancements';
+export const JOBS_API = {
+  DELETE: `${BASE_QUERY_ENHANCEMENTS_API}/jobs`,
+};
+
+export const INDEX_WITH_TIME_1 = 'data_logs_small_time_1';
```

Comment:
this seems redundant since i think we reference the same stuff from the discover tests

Reply:
just fyi, all `utils` for enhancements have been copied and pasted to have explore counterpart. In my initial Cypress test addition we initially only branched out of enhancement specific utils when we needed to, but then as we progressed it became very hard to manage, as further utils needed to be branched out and it was proving difficult to keep track.

We should consolidate them but i believe this should be done separately, instead of sweating and monkeying around the code half-hazardly when you are working on a feature